libs/components/indicators/testing/src/tokens/token-harness.ts

```diff
@@ -34,6 +34,17 @@ export class SkyTokenHarness extends ComponentHarness {
     );
   }
 
+  /**
+   * Selects the token.
+   */
+  public async select(): Promise<void> {
+    if (await (await this.#getWrapper()).hasClass('sky-btn-disabled')) {
```

Comment:
Is there a public method we give to consumers to check if the token is disabled? If not, should we add that method to this PR?

Reply:
I added an `isDisabled()` method to the token harness. There's no indication in the DOM whether the tokens component is disabled, so I didn't add the same method to it.